_sass/_footer.scss

```
@@ -0,0 +1,120 @@
+/* Footer */
+footer.footer-global {
+  padding: 15px;
+  color: #898989;
+  font-weight: 200;
+	font-size: 17px;
+  background: rgb(17, 17, 17);
+  @media (prefers-color-scheme: light) {
+    background: #fff;
+    color: #2c2c2c;
+  }
+  @media (max-width: 768px) {
+    padding: 23px;
+  }
+  .wrapper {
+    max-width: 1210px;
+    margin: 0 auto;
+    @media (min-width: 768px) {
+      margin: 60px auto;
+    }
+  }
+  hr {
+    border: none;
+    margin: 50px 0;
+    border-top: 1px solid #2c2c2c;
+    @media (prefers-color-scheme: light) {
+      border-top: 1px solid #e7e7e7;
+    }
+    @media (max-width: 768px) {
+      margin: 30px 0 50px;
+    }
+  }
+	a {
+		color: inherit;
+		text-decoration-color: #898989;
+		text-decoration-thickness: 1px;
+    transition: all 0.2s ease-in-out;
+    &:hover {
+      color: var(--link-color);
+      text-decoration-color: var(--link-color);
+    }
+    img {
+      display: inline-block;
+    }
+	}
+  h2 {
+    font-size: 25px;
+    margin-bottom: 14px;
+    color: white;
+    @media (prefers-color-scheme: light) {
+      color: #2c2c2c;
+    }
+    @media (max-width: 768px) {
+      font-size: 20px;
+    }
+  }
+  ul {
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+    margin-bottom: 30px;
+    li {
+      margin-bottom: 5px;
+      font-size: 16px;
+    }
+    a {
+      text-decoration: none;
+    }
+  }
+  .columns {
+    display: grid;
+    grid-template-columns: 1fr 1fr 1fr 1fr;
+    @media (max-width: 768px) {
+      grid-template-columns: 1fr;
+    }
+  }
+  .credits-and-socials {
+    display: grid;
+    align-items: center;
+    grid-template-columns: 1fr 1fr;
+    @media (max-width: 768px) {
+      grid-template-columns: 1fr;
+      text-align: center;
+    }
+    p {
+      margin: 0;
+      max-width: 500px;
+      @media (max-width: 768px) {
+        margin: 0 auto;
+      }
+    }
+    .social {
+      text-align: right;
+      @media (max-width: 768px) {
+        margin-top: 30px;
+        text-align: center;
+      }
+      svg {
+        max-width: 32px;
+        max-height: 32px;
+        transition: all 0.2s ease-in-out;
+        fill: white;
+        opacity: 0.5;
+        @media (prefers-color-scheme: light) {
+          fill: #000;
+        }
+        &:hover {
+          fill: var(--link-color);
+          opacity: 1;
+        }
+      }
+      a {
+        text-decoration: none;
+        margin-right: 9px;
+        display: inline-block;
+        width: 32px;
+      }
+    }
+  }
+}
```

Add newline at end of file:

```suggestion
}

```